### PR TITLE
[6.15.z] Add Capsule Provisioning test

### PR DESCRIPTION
**Pull request by Satellite-QE**
Cherrypick of PR: https://github.com/SatelliteQE/robottelo/pull/13241

Adding a test for provisioning using an external capsule.

### CODE REVIEW

**Comment by Satellite-QE** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_provisioning.py -k test_capsule_pxe_provisioning
provisioning: true
pod_resources_size: extra_extra_large

**Comment by Gauravtalreja1** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_provisioning.py -k test_capsule_pxe_provisioning
provisioning: true
pod_resources_size: extra_extra_large

File: pytest_fixtures/component/provision_capsule_pxe.py [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+    capsule_provisioning_sat,
+    module_sca_manifest_org,
+    module_lce_library,
+):
+    """
+    This fixture sets up kickstart repositories for a specific RHEL version
+    that is specified in `request.param`.
+    """
+    sat = capsule_provisioning_sat.sat
+    rhel_ver = request.param
```
Comment (Gauravtalreja1) [inline pytest_fixtures/component/provision_capsule_pxe.py:176]:
```suggestion
    rhel_ver = request.param['rhel_version']
```

**Comment by Gauravtalreja1** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_provisioning.py -k test_capsule_pxe_provisioning
provisioning: true
pod_resources_size: extra_extra_large

**Review by Gauravtalreja1 — APPROVED** [review]
Ack, EL7 provisioning failures look unrelated

**Pull request merged**